### PR TITLE
feat(oidc): allow service worker to not hide acces_token

### PR DESCRIPTION
**Pull request by guillaume-chervet**
Feature for #1038.

Need to add unit test.

### CODE REVIEW

File: packages/react/public/types.d.ts [hunk 1]
```diff
@@ -47,6 +50,7 @@ export type OidcConfig = {
     oidcConfiguration?: OidcConfiguration;
     sessionState?: string | null;
     items?: MessageData;
+    hideAccessToken: boolean;
```
Comment (darkowic) [inline packages/react/public/types.d.ts:53]: `hideAccessToken` is `true` by default. Should we call this the opposite: `showAccessToken`?

File: packages/react/service_worker/OidcServiceWorker.ts [hunk 2]
```diff
@@ -274,11 +279,14 @@ const handleMessage = (event: ExtendableMessageEvent) => {
       nonce: null,
       status: null,
       configurationName,
+      hideAccessToken: trustedDomainsHideAccessToken[configurationName],
```
Comment (darkowic) [inline packages/react/service_worker/OidcServiceWorker.ts:282]: @krzempekk we need to evaluate this.

@guillaume-chervet we need to check if the trusted domains are something we can dynamically generate based on our deployment target. In our deployment model, we follow the: build once and deploy to many environments (dev/staging/prod). Therefore, anything hardcoded at the build time is a problem.

Comment (guillaume-chervet) [reply packages/react/service_worker/OidcServiceWorker.ts:282]: Thank you for your feedback @darkowic.

I'am waiting for your response :)

Comment (HSULGIT) [reply packages/react/service_worker/OidcServiceWorker.ts]: How can we set the access token to be shown?

**Pull request merged**